gradio/components/line_plot.py

```diff
@@ -415,10 +415,10 @@ def postprocess(self, y: pd.DataFrame | dict | None) -> dict[str, str] | None:
             tooltip=self.tooltip,
             x_title=self.x_title,
             y_title=self.y_title,
-            color_legend_title=self.color_legend_title,
-            color_legend_position=self.color_legend_position,
+            color_legend_title=self.color_legend_title,  # type: ignore
```

Comment:
I'm not sure why pyright is inferring these as `str` instead of `Literal` could be a bug